packages/cursorless-engine/src/actions/Sort.ts

```diff
@@ -1,33 +1,31 @@
 import { shuffle } from "lodash";
 import { Target } from "../typings/target.types";
-import { Graph } from "../typings/Graph";
+import { Actions } from "./Actions";
 import { Action, ActionReturnValue } from "./actions.types";
 
 abstract class SortBase implements Action {
-  constructor(private graph: Graph) {
+  constructor(private actions: Actions) {
     this.run = this.run.bind(this);
   }
 
   protected abstract sortTexts(texts: string[]): string[];
 
-  async run(targets: Target[][]): Promise<ActionReturnValue> {
+  async run([targets]: [Target[]]): Promise<ActionReturnValue> {
```

Comment:
For some reason typescript started complaining about this one, so I fixed it up a bit; note that there's no reason to support arbitrary number of target lists